projects/spectator/test/events/events.component.spec.ts

```diff
@@ -39,6 +39,16 @@ describe('EventsComponent', () => {
     expect(spectator.query('h1')).toHaveText('pressed dot');
   });
 
+  it('should include key and keyCode when KeyboardEventOptions are passed', () => {
```

Comment:
Update the docs, please.

Reply:
I've updated the docs where `dispatchKeyboardEvent` was referenced. Please let me know if this is sufficient.

Reply:
I've updated the docs with a concrete example.